tests/integration-tests/tests/update/test_update.py

```diff
@@ -837,6 +841,36 @@ def _test_update_queue_strategy_with_running_job(
     assert_compute_node_states(scheduler_commands, queue1_nodes, "idle")
 
 
+@pytest.fixture
+@pytest.mark.usefixtures("parameterized_cfn_stacks_factory")
+def external_shared_storage_stack(request, test_datadir):
+    def _create_stack(region, vpc, PublicAz3, PublicAz2, import_path, export_path, Public):
+        template_path = os.path.join(str(test_datadir), "storage-stack.json")
+        option = "external_shared_storage_stack_name"
+        if request.config.getoption(option):
+            logging.info("Using stack {0} in region {1}".format(request.config.getoption(option), region))
+            stack = parameterized_cfn_stacks_factory(name=request.config.getoption(option), region=region, template_path=template_path)
+        else:
+            params = [
+                {"ParameterKey": "vpc", "ParameterValue": vpc},
+                {"ParameterKey": "PublicAz3", "ParameterValue": PublicAz3},
+                {"ParameterKey": "PublicAz2", "ParameterValue": PublicAz2},
+                {"ParameterKey": "import_path", "ParameterValue": import_path},
+                {"ParameterKey": "export_path", "ParameterValue": export_path},
+                {"ParameterKey": "Public", "ParameterValue": Public},
+            ]
+            stack = parameterized_cfn_stacks_factory(
+                stack_prefix="integ-tests-external-shared-storage",
+                region=region,
+                parameters=params,
+                template_path=template_path,
+                capabilities=["CAPABILITY_IAM"]
```

Comment:
This line may introduce lint errors due to missing comma. Have a look at GH checks for the PR